migrations/2021-02-10-141019_reserve_com0_lpt0/up.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO reserved_crate_names (name) VALUES ('com0', 'lpt0') ON CONFLICT DO NOTHING;
```

Comment:
```suggestion
INSERT INTO reserved_crate_names (name) VALUES ('com0'), ('lpt0') ON CONFLICT DO NOTHING;
```

Reply:
Pushed a fix for this as soon as you commented :sweat_smile: